### ISSUE TITLE
No way to specify the timezone while reading from redshift

### ISSUE THREAD

**Issue by Gauravshah** [databricks/spark-redshift#184]
redshift doesn't support timezones. So the times are stored without timezone.

If a time is stored in PST format in redshift, then it loads incorrectly in spark-redshift. 

spark-redshift assumes that the time is in UTC

**Comment by JoshRosen** [comment]
Given that the timezone isn't encoded anywhere in the table metadata or the stored values, I don't see any clear way for us to automatically support the right timezone. I suppose that a table could technically have multiple time columns in different timezones, so if we added a configuration for this I guess it would have to be settable on a per-column basis.

Would it work for your user-level / application-level to add a time-zone offset to the time columns after reading them? In other words, can this be worked around cleanly without requiring changes to `spark-redshift` itself?

**Comment by Gauravshah** [comment]
Yes currently I am doing that , it does work. Logically one would not have multiple columns of same table in different timezones so a table one would help, but nevermind.

Also writing data back has similar issues. It would be helpful if you can add it to the readme that it converts/assumes in such a way.

Thanks

**Comment by Gauravshah** [comment]
The issue happens especially when we write the data. For eg:
`to_date(from_utc_timestamp(from_unixtime(1457078400), 'America/Los_Angeles'))`
get written as March 3 2016, but its expected to store March 4 2016

`1457078400=> Fri, 04 Mar 2016 08:00:00 GMT`
`from_utc_timestamp(from_unixtime(1457078400), 'America/Los_Angeles') = 3/4/2016, 12:00:00 AM`
